x/ccv/provider/keeper/keeper.go

```diff
@@ -698,10 +698,10 @@ func (k Keeper) GetPendingVSCs(ctx sdk.Context, chainID string) (packets []ccv.V
 
 // AppendPendingVSC adds the given ValidatorSetChange packet to the list
 // of pending ValidatorSetChange packets stored under chain ID
-func (k Keeper) AppendPendingVSC(ctx sdk.Context, chainID string, packet ccv.ValidatorSetChangePacketData) {
+func (k Keeper) AppendPendingVSC(ctx sdk.Context, chainID string, newPackets []ccv.ValidatorSetChangePacketData) {
```

Comment:
Maybe rename to `AppendPendingVSCs` sice `newPackets` is a slice of VSC packets and keep it consistent with other methods (`GetPendingVSCs`, `DeletePendingVSCs`)

Reply:
It's also a good place to use a  variadic function:
```
// AppendPendingVSC takes an arbitrary number of `ValidatorSetChangePacketData`
func (k Keeper) AppendPendingVSC(ctx sdk.Context, chainID string, newPackets ...ccv.ValidatorSetChangePacketData) {}

k.AppendPendingVSCs(ctx, chainID, cs.PendingValsetChanges...)

k.AppendPendingVSCs(ctx, chainID, packetData)
```